index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Registration Form</title>
+    <link rel="stylesheet" href="css/style.css">
+    
+  </head>
+
+  <body>
+    <header>
+      <h1 id="title">Microverse Registration Form</h1>
+      <p id="description">Please register your information</p>
+    </header>
+      <form id="suervey-form">          
```

Comment:
- [x] You have done a very good job requesting this change, however, Please use kind words, be clear, and always give a clear explanation of why the requested change is necessary.

for example, you could say: :arrow_down: 

- [ ] Good job so far, :clap: However, Please ensure that you align your child element properly, child elements are typically placed within the parent element and rendered in the order they appear in the HTML markup. By default, child elements will flow one after another vertically within the parent element, unless you specify a different layout. Please reslve it. :slightly_smiling_face: